pkg/controller/remoteingress/remoteingress_controller_test.go

```diff
@@ -362,6 +364,26 @@ func TestRemoteClusterIngressReconcile(t *testing.T) {
 				fmt.Sprintf("%s-secondCertBundle-secret", testClusterName),
 			},
 		},
+		{
+			name: "Test setting httpErrorCodePages",
+			localObjects: func() []runtime.Object {
+				objects := []runtime.Object{}
+				cd := addIngressToClusterDeployment(testClusterDeployment(), "secondingress", "moreingress.example.com", nil, nil, "", configv1.ConfigMapNameReference{})
+				cd.Status.Conditions = append(cd.Status.Conditions, hivev1.ClusterDeploymentCondition{
+					Type:   hivev1.IngressCertificateNotFoundCondition,
+					Status: corev1.ConditionUnknown,
+				})
+				objects = append(objects, cd)
+				return objects
+			}(),
+			expectedSyncSetIngressEntries: []SyncSetIngressEntry{
+				{
+					name:               testDefaultIngressName,
+					domain:             testIngressDomain,
+					HttpErrorCodePages: testHttpErrorCodePages(),
```

Comment:
...and this isn't failing the test because we need an update to the `validateSyncSet` function. It explicitly checks each of the fields in each of the ingress controller entries; we need to add a line to check the HttpErrorCodePages.